election-day/obetsun/election_day.go

```diff
@@ -0,0 +1,38 @@
+package electionday
+
+import "strconv"
+
+// NewVoteCounter returns a new vote counter with
+// a given number of inital votes.
+func NewVoteCounter(initialVotes int) *int {
+	var counter int
```

Comment:
Something is breaking the indentation in your IDE. I see it here, in the next function, on line 26 and I remember seeing this on your MRs on the service repo. Get it fixed or take it to a doctor :-).
And you do not need the extra var here, just return the address of initialVotes.
Other than that it's all nice!

Reply:
I think the indentation here is from vim-go plugin I use, it always replaces spaces with tabs.